test/controllers/beneficial.owner.gov.controller.spec.ts

```diff
@@ -29,7 +29,7 @@ import {
   PAGE_TITLE_ERROR,
   SAVE_AND_CONTINUE_BUTTON_TEXT,
   SERVICE_UNAVAILABLE,
-  SHOW_INFORMATION_ON_PUBLIC_REGISTER
+  SHOW_INFORMATION_ON_PUBLIC_REGISTER, UK_SANCTIONS_DETAILS
```

Comment:
Small one, could you please add `UK_SANCTIONS_DETAILS` on a new line.
Same in the other two files.

Reply:
Moved to a new line.